// @progress/kendo-layout-vue-wrapper/dist/npm/KendoTabStrip/KendoTabStripMixin.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _kendoBaseComponentsVueWrapper = require('@progress/kendo-base-components-vue-wrapper');

var _TabStripProps = require('./TabStripProps');

var _TabStripProps2 = _interopRequireDefault(_TabStripProps);

var _Animation = require('../Common/Animation');

var _Animation2 = _interopRequireDefault(_Animation);

var _Scrollable = require('../Common/Scrollable');

var _Scrollable2 = _interopRequireDefault(_Scrollable);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
    props: _TabStripProps2.default,
    render(h) {
        return h('div', this.$slots.default);
    },
    mixins: [_kendoBaseComponentsVueWrapper.KendoBaseComponent, _kendoBaseComponentsVueWrapper.KendoBaseDatasourceComponent, _Animation2.default, _Scrollable2.default],
    mounted() {
        kendo.jQuery(this.$el).kendoTabStrip(this.widgetOptions);

        this.$emit('kendowidgetready', this.kendoWidget());
    },
    methods: {
        kendoWidget() {
            return kendo.jQuery(this.$el).getKendoTabStrip();
        }
    }
};
