// @progress/kendo-layout-vue-wrapper/dist/npm/KendoContextMenu/KendoContextMenuMixin.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _kendoBaseComponentsVueWrapper = require('@progress/kendo-base-components-vue-wrapper');

var _ContextMenuProps = require('./ContextMenuProps');

var _ContextMenuProps2 = _interopRequireDefault(_ContextMenuProps);

var _Animation = require('./Animation');

var _Animation2 = _interopRequireDefault(_Animation);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
    render(h) {
        return h('ul', this.$slots.default);
    },
    props: _ContextMenuProps2.default,
    mixins: [_kendoBaseComponentsVueWrapper.KendoBaseComponent, _kendoBaseComponentsVueWrapper.KendoSharedMethods, _Animation2.default],
    mounted() {
        this._resolveChildren();

        kendo.jQuery(this.$el).kendoContextMenu(this.widgetOptions);

        this.$emit('kendowidgetready', this.kendoWidget());
    },
    methods: {
        kendoWidget() {
            return kendo.jQuery(this.$el).getKendoContextMenu();
        },
        _resolveChildren() {
            if (this.$options.propsData && !this.$options.propsData['dataSource'] && this.$slots.default) {
                var items = this.resolveInnerTags('kendo-menu-item');

                if (items.length) {
                    this.widgetOptions['dataSource'] = items;
                }
            }
        }
    }
};
