// @progress/kendo-layout-vue-wrapper/dist/npm/KendoPanelBar/KendoPanelBarMixin.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _kendoBaseComponentsVueWrapper = require('@progress/kendo-base-components-vue-wrapper');

var _PanelBarProps = require('./PanelBarProps');

var _PanelBarProps2 = _interopRequireDefault(_PanelBarProps);

var _Animation = require('./Animation');

var _Animation2 = _interopRequireDefault(_Animation);

var _Messages = require('./Messages');

var _Messages2 = _interopRequireDefault(_Messages);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
    props: _PanelBarProps2.default,
    render(h) {
        return h('ul', this.$slots.default);
    },
    mixins: [_kendoBaseComponentsVueWrapper.KendoBaseComponent, _kendoBaseComponentsVueWrapper.KendoBaseDatasourceComponent, _kendoBaseComponentsVueWrapper.KendoSharedMethods, _Animation2.default, _Messages2.default],
    mounted() {
        this._resolveChildren();

        kendo.jQuery(this.$el).kendoPanelBar(this.widgetOptions);

        this.$emit('kendowidgetready', this.kendoWidget());
    },
    methods: {
        kendoWidget() {
            return kendo.jQuery(this.$el).getKendoPanelBar();
        },
        _resolveChildren() {
            if (this.$options.propsData && !this.$options.propsData['dataSource'] && this.$slots.default) {
                var items = this.resolveInnerTags('kendo-panelbar-item');

                if (items.length) {
                    this.widgetOptions['dataSource'] = items;
                }
            }
        }
    }
};
